pkgs/applications/networking/protonmail-bridge/default.nix

```diff
@@ -55,7 +55,7 @@ in mkDerivation {
       --set-rpath "${rpath}" \
       $out/lib/protonmail-bridge
 
-    substituteInPlace $out/share/applications/ProtonMail_Bridge.desktop \
+    substituteInPlace $out/share/applications/protonmail_bridge.desktop \
```

Comment:
I can't build the package, there is no `protonmail_bridge.desktop` file, but there is a `protonmail-bridge.desktop` file.